### PR TITLE
Changed server_id size from 3 to 17

### PR DESCRIPTION
**Pull request by tperalta82**
Schema change due to  #99

### CODE REVIEW

**Comment by aik099** [comment]
Rebasing and squashing is needed.

File: xhprof_lib/utils/Db/Mssql.php [hunk 1]
```diff
@@ -1,46 +1,46 @@
...
+ [server name] nvarchar(64) NULL DEFAULT NULL,
+ perfdata nvarchar(max) NULL,
+ type smallint NULL DEFAULT NULL,
+ cookie nvarchar(max) NULL,
+ post nvarchar(max) NULL,
+ get nvarchar(max) NULL,
+ pmu int NULL DEFAULT NULL,
+ wt int NULL DEFAULT NULL,
+ cpu int NULL DEFAULT NULL,
+ server_id nchar(17) NOT NULL DEFAULT N't11',
```
Comment (aik099) [inline xhprof_lib/utils/Db/Mssql.php]: All lines appear as changed. Probably you've made accidental mass-whitespace changes, that needs to be removed.

Same goes for other made changes (SQL & PHP).

Comment (aik099) [inline xhprof_lib/utils/Db/Mssql.php]: The problem comes from mass-replacement of line endings from `\r\n` into `\n` it seems.

**Comment by tperalta82** [comment]
Yeah I noticed, it doesn't make any sense at all, before pushing &
commiting , I tried with different editors, from gedit to pluma to vi /
nano / netbeans, something is reallty wrong , well, reject all the pr's ...
again, as this doesn't make any sense.

I'll redo this when I have the time to reset the repo and start the
modifications from scratch

**Comment by aik099** [comment]
PR does make sense, but it needs to be fixed. Fixing isn't really that hard. No need to close PR and sending it again. Just change existing PR branch.

**Comment by flip111** [comment]
@tperalta82 your PR looks good. I'm running into this issue now too.

**Comment by aik099** [comment]
This was done in #122.

**Pull request closed without merging**